Mathlib/SetTheory/ZFC/Basic.lean

```diff
@@ -557,6 +557,7 @@ def Resp.Equiv {n} (a b : Resp n) : Prop :=
   Arity.Equiv a.1 b.1
 #align pSet.resp.equiv PSet.Resp.Equiv
 
+@[refl]
```

Comment:
This is a weird way to spell `symm`! Please mention there's a `refl` attribute too in the PR description.